### ISSUE TITLE
Feature Request: Filter out posts by specific authors

### ISSUE THREAD

**Issue by asmaloney** [ViennaRSS/vienna-rss#1454]
**Is your feature request related to a problem? Please describe.**

I subscribe to feeds for blogs with several authors. Some authors are... better than others. I'd rather not see posts in the feed from some authors.

**Describe the solution you would like**

For each feed it would be nice to be able to specify a filter so posts from specific authors don't show up. Maybe just a simple comma-separated list of names?

"Ignore posts from these authors:" [Some Author, Some Other Author]

**Comment by asmaloney** [comment]
Ah - I found "Smart Folders". I was looking for things related to "feed" not "folder".

So I tried a "smart folder" with "match all", "folder set to the feed I want, and "Author" "is not" "Foo Bar"... and I get an empty list of articles. Remove the "Author" one and the articles show up again.

The feed uses `<dc:creator>` and the authors show up properly in the "un-smart folder".

macOS 10.14.6
Version 3.7.1 :d356d926: (7437)
